README.md

```diff
@@ -1,2 +1,42 @@
 # cmc-ccd-submit-domain
-save a civil money claim in ccd - creation domain
+
+Repo for shared CMC/CCD Java model and CMC/CCD Definitions. It will contain:
+
+* CMC CCD model
+* CMC CCD JSON Definition
+* Tooling to release/manage CCD Definition
+* When merging into master:
+    * Release the Docker CCD Definition image
+    * Release new JAR
+    * Exercise cmc-claim-store and cmc-claim-submit-api functional tests (and any other consumers)
+
+## Master
+
+Currently any merge/commit to master will trigger an ACR task to build CCD Definition Docker image. See /definitions for more details.
+
+## Development
+
+1. Open PR here, bumping up /definitions/VERSION.yaml to 1.2.1 (no functional test running at this stage)
+    * cmc-ccd-domain - master 1.2.0
+    * cmc-claim-store - master 1.2.0
+    * cmc-integration-tests - master 1.2.0
+1. Merge PR
```

Comment:
you might want to correct the numbering...

Reply:
this is markdown formatting 😊 so you dont have to manually think about numbers - you can see how rendered in view file